toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/ViewpointPersistence.kt

```diff
@@ -17,23 +17,42 @@
 
 package com.arcgismaps.toolkit.geocompose
 
+import android.os.Parcel
+import android.os.Parcelable
+
 /**
  * Enum class representing the different types of viewpoint persistence on a composable [MapView].
  *
  * Viewpoint persistence determines how the viewpoint of a MapView is saved and restored across activity
  * or process recreation, for example, when the device is rotated or when the app is sent to the background
  * and then brought back to the foreground.
  *
+ * If the [ViewpointPersistence] is implemented with a [MutableState] type, it must be wrapped in [rememberSaveable],
+ * otherwise the state will not be saved and restored across activity or process recreation and the
+ * behavior of the [MapView] will be unpredictable.
+ *
  * @since 200.4.0
  */
-public sealed class ViewpointPersistence {
+public sealed class ViewpointPersistence : Parcelable {
 
     /**
      * The viewpoint is not persisted.
      *
      * @since 200.4.0
      */
-    public object None : ViewpointPersistence()
+    public object None : ViewpointPersistence() {
+        override fun describeContents(): Int = 0
+
+        override fun writeToParcel(dest: Parcel, flags: Int) {
+            // No state to write for this object
+        }
+
+        @JvmField
+        public val CREATOR: Parcelable.Creator<None> = object : Parcelable.Creator<None> {
```

Comment:
Should it be a nullable type parameter?
```suggestion
        public val CREATOR: Parcelable.Creator<None?> = object : Parcelable.Creator<None?> {
```